### DISCUSSION TITLE
Italian Geology Site

### DISCUSSION THREAD

**Discussion by adhiabulzz** [eliemichel/MapsModelsImporter#171] [Ideas]
Hello!
I've been trying around with the mapmodelsimporter and it was great! the capture merger and texture packer is also very useful.
I was wondering is it possible to add support for this site?
http://www.pcn.minambiente.it/viewer3D/
There are no 3D buildings there, but it got the highest resolution DEM for the terrain around Italian region.
I'm making road tracks for racing sim, and a good DEM would absolutely be helpful.

Many thanks!

**Comment by Ben-Cal** [comment]
Hey! I tried it to and couldn't get the importer to work before and couldn't, the capturing seems to work in renderdoc but the import doesn't. It would be great to make the import possible.

Thank you so much for all the work, for Google Maps it works perfectly and Lily Merger is absolutely much more than worth the money!
